fabric/build.gradle

```diff
@@ -10,6 +10,9 @@ final BuildTools buildTools = new BuildTools(minecraftVersion, "fabric", project
 
 loom {
 	accessWidenerPath = file("src/main/resources/mtr.accesswidener")
+	runConfigs.configureEach {
+		ideConfigGenerated = true
+	}
```

Comment:
I couldn't get the generated Fabric client run config to work in Intellij, did you have any success? I am removing this for now.

Reply:
It works here (At least in the making of this PR), although IIRC I need to restart IntelliJ for it to not crash on launch

If that doesn't work it's fine if we remove this for now

Reply:
Deleting the `.idea` folder refreshed the cache and did the trick, thanks!